### DISCUSSION TITLE
Unable to locate a class or view for component - heroicons

### DISCUSSION THREAD

**Discussion by nettsite** [filamentphp/filament#6889] [General]
I can't find anything on this here, so maybe I am bad at searching, or I am doing something wrong.

I would like to specify icons for items in the menu, and I am doing it like so in the resource:

` protected static ?string $navigationIcon = 'heroicon-o-building-office-2';`

I found that icon on [https://blade-ui-kit.com/blade-icons/heroicon-o-building-office-2](https://blade-ui-kit.com/blade-icons/heroicon-o-building-office-2)

I get the following error:

`Unable to locate a class or view for component [heroicon-o-building-office-2].`

Some other icons work fine, e.g. 

`protected static ?string $activeNavigationIcon = 'heroicon-s-document-text';`

How do I know which icons I can use, or even better, how do I add icons to the useable library?

OR, should I just not worry about this until version 3 is available?

**Comment by danharrin** [comment]
You need to use Heroicons v1 (https://v1.heroicons.com), Heroicons v2 are available in Filament v3.

**Reply by nettsite** [reply]
Thank you!
